pyiron_atomistics/atomistics/structure/neighbors.py

```diff
@@ -70,6 +71,36 @@ def __init__(self, ref_structure):
         self.cutoff_radius = np.inf
         self._norm_order = 2
 
+    @property
+    def mode(self):
+        """
+        Change the mode of representing attributes (vecs, distances, indices, shells). The shapes
+        of `filled` and `ragged` differ only if `cutoff_radius` is specified.
+
+        - 'filled': Fill variables for the missing entries are filled as follows: `np.inf` in
+            `distances`, `numpy.array([np.inf, np.inf, np.inf])` in `vecs`, `n_atoms+1` (or a
+            larger value) in `indices` and -1 in `shells`.
+
+        - 'ragged': Create lists of different lengths.
+
+        - 'flattened': Return flattened arrays for distances, vecs and shells. The indices
+            corresponding to the row numbers in 'filled' and 'ragged' are in `atom_numbers`
+
+        The variables are stored in the `filled` mode.
+        """
+        for k,v in self._mode.items():
+            if v:
+                return k
+
+    @mode.setter
+    def mode(self, new_mode):
+        if new_mode not in self._mode.keys():
+            raise KeyError(
+                new_mode + " not found. Available modes: 'filled', 'ragged', 'flattened'"
+            )
+        self._mode = dict.fromkeys(list(self._mode.keys()), False)
```

Comment:
Can be simplified as below no?

```suggestion
        self._mode = {key: False for key in self._mode.keys()}
```